streaming_jupyter_integrations/magics.py

```diff
@@ -380,11 +380,13 @@ def __interrupt_execute(self, *args: Any) -> None:
 
     def __handle_done(self, fut: Any) -> None:
         self.background_execution_in_progress = False
-        print("Execution done")
         # https://stackoverflow.com/questions/48161387/python-how-to-print-the-stacktrace-of-an-exception-object-without-a-currently
         # will raise an exception to the main thread
```

Comment:
```suggestion
        # https://stackoverflow.com/questions/48161387/python-how-to-print-the-stacktrace-of-an-exception-object-without-a-currently
```

Please remove the second line of that comment, as it is not true from now on :)

Reply:
done